python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -400,27 +385,27 @@ async def health_monitor(self):
             if self.summary_state == salobj.State.ENABLED:
                 if self.seeing_loop_task.done():
                     error_report = "Seeing loop died while in enable state."
-                    self.evt_errorCode.set_put(
+                    await self.evt_errorCode.set_write(
```

Comment:
Please don't set or write evt_errorCode yourself. BaseCsc.fault does that.

Please look for and remove this everywhere.

Reply:
Thanks. I found two instances of writing that error code and removed both.